_topic_map.yml

```diff
@@ -509,6 +509,8 @@ Topics:
   File: migrating_etcd
 - Name: Downgrading
   File: downgrade
+- Name: Upgrading When Using GlusterFS Container Native Storage
```

Comment:
Do not use `GlusterFS Container Native Storage` to refer to GlusterFS. Please use the phrase `Containerized GlusterFS`.